### PR TITLE
Update Set.difference signature for 1.21.0-dev.2.0

### PR DESCRIPTION
**Pull request by keertip**
@kevmoo

### CODE REVIEW

**Comment by kevmoo** [comment]
Didn't this just land in fcdbd1c – 1.21.0-dev.2 ?

File: pubspec.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name: html
-version: 0.13.0
+version: 0.14.0
```
Comment (kevmoo) [inline pubspec.yaml]: Fine to do a +1 release here – no other breaking changes. Folks won't be able to get this without bumping their SDK – let's avoid a cascading dependency dance.

File: pubspec.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 author: Dart Team <misc@dartlang.org>
 description: A library for working with HTML documents. Previously known as html5lib.
 homepage: https://github.com/dart-lang/html
 environment:
-  sdk: '>=1.2.0 <2.0.0'
+  sdk: '>=1.21.0-dev.1.0 <2.0.0'
```
Comment (kevmoo) [inline pubspec.yaml]: As I said – fcdbd1c – this is `1.21.0-dev.2.0` – right?

File: pubspec.yaml (hunk 1 above)
Comment (keertip) [inline pubspec.yaml]: changed to 0.13.1

File: pubspec.yaml (hunk 2 above)
Comment (keertip) [inline pubspec.yaml]: done

**Comment by keertip** [comment]
PTAL, updated changelog too.

**Pull request merged**